src/cmd/linuxkit/pkglib/docker.go

```diff
@@ -5,16 +5,32 @@ package pkglib
 //go:generate ./gen
 
 import (
+	"bytes"
+	"encoding/base64"
 	"fmt"
 	"io"
 	"os"
 	"os/exec"
+	"path"
+	"strings"
 
+	"github.com/docker/cli/cli/config"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/sync/errgroup"
 )
 
-const dctEnableEnv = "DOCKER_CONTENT_TRUST=1"
+const (
+	dctEnableEnv                     = "DOCKER_CONTENT_TRUST=1"
+	registry                         = "https://index.docker.io/v1/"
+	notaryServer                     = "https://notary.docker.io"
+	notaryDelegationPassphraseEnvVar = "NOTARY_DELEGATION_PASSPHRASE"
+	notaryAuthEnvVar                 = "NOTARY_AUTH"
+	dctEnvVar                        = "DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE"
+)
+
+var platforms = []string{
```

Comment:
This is currently only used in the invocation of `manifest-tool` and could be a constant string, comma separated.

Reply:
It could, but since it describes all of our supported platforms, I suspect we will either consume it from elsewhere or provide it to elsewhere, so I kept it independent.